### PR TITLE
Documentation updates for react-native

### PR DESCRIPTION
**Pull request by chromakode**
A few updates for steps I had to take to get the react-native codebase up and running.

:eyeglasses: @keybase/react-hackers

### CODE REVIEW

File: react-native/README.md [hunk 1]
```diff
@@ -42,6 +45,12 @@ npm run watch
 
 #### iOS
 
+Install cocoapods:
+
+```sh
+sudo gem install cocoapods --pre -v 1.0.0.beta.6
```
Comment (gabriel) [inline react-native/README.md]: if you use rbenv you don't have to sudo but i guess most people probably use the default os x ruby install so...

**Comment by gabriel** [comment]
:shipit:

File: react-native/README.md (hunk 1 above)
Comment (chromakode) [inline react-native/README.md]: Yeah I personally used `--user-install` but didn't want to have to include the details. Could drop the sudo to match the `npm install -g` and let the user figure it out :smirk:

File: react-native/README.md [hunk 2]
```diff
@@ -21,13 +21,16 @@ We are iterating quickly and a lot of the code is changing every day.
...
+Follow the [React Native Getting Started guide](https://facebook.github.io/react-native/docs/getting-started.html) for the platforms you intend to develop on.
+
 ```sh
 # Setup golang mobile
 go get golang.org/x/mobile/cmd/gomobile
 gomobile init
 
 # Setup npm
 npm install
+npm-install -g react-native-cli
```
Comment (chrisnojima) [inline react-native/README.md]: i don't think this is usually necessary

Comment (chromakode) [inline react-native/README.md]: @chrisnojima this was necessary for the `react-native run-android` command to work. I tried running `~/node_modules/.bin/react-native` after my initial npm install, but it was just a placeholder that instructed me to install `react-native-cli`.

**Comment by chromakode** [comment]
https://github.com/keybase/client/commit/0e6ce0a35d711719489a5806b457bb8acb6efd1f

**Pull request closed without merging**